### ISSUE TITLE
NTR: Treponema pallidum in blood PCR positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8061]
New term request Treponema pallidum in blood PCR positive (samant_101126232142): Polymerase chain reaction; syphilis; DNA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000545
- label: Positive treponema pallidum PCR test in the blood circulation
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: PMID:35315702
- def: Detection of nucleic acid of treponema pallidum in the blood circulation by a method such a polymerase chain reaction.
- comment: Treponema pallidum is the causative agent of syphilis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8061
- synonyms:
